src/components/TodoComp.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+  <div>
+    <TodoTitle v-if="!isEditing" :done="todo.done" :title="todo.title" />
+
+    <InputBox
+      v-if="isEditing"
+      identifier="initalInput"
+      :value="editText"
+      v-model="editText"
```

Comment:
You are using v-model. It can be used as  props `:value` in the child component. No need for additional `:value` here